presidio-analyzer/presidio_analyzer/predefined_recognizers/phone_recognizer.py

```diff
@@ -21,20 +21,15 @@ class PhoneRecognizer(LocalRecognizer):
 
     SCORE = 0.6
     CONTEXT = ["phone", "number", "telephone", "cell", "cellphone", "mobile", "call"]
-    DEFAULT_SUPPORTED_COUNTRY_CODES = ("US", "UK", "DE", "FE", "IL")
 
     def __init__(
         self,
         context: Optional[List[str]] = CONTEXT,
         supported_language: str = "en",
-        supported_entities: List[str] = [
-            code + ENTITY_TYPE_SUFFIX for code in DEFAULT_SUPPORTED_COUNTRY_CODES
-        ],
-        support_international=True
+        supported_regions=SUPPORTED_REGIONS
```

Comment:
@omri374 By default, I thought about supporting all regions in phonenumbers. The results were pretty fast still and we are removing duplicates anyways. WDYT?

Reply:
Eventually decided to return to the hardcoded list. 
Japanese number was matched with credit card, Mongolian with US driving licence, and these are just the tests we have.